### ISSUE TITLE
access to card comments?

### ISSUE THREAD

**Issue by kevinchugh** [jeremytregunna/ruby-trello#106]
Hi, is this possible? I don't see any method or attribute for card comments.  Thanks for any info.

**Comment by Larusso** [comment]
You can post a comment with `add_comment text` to a card. It is possible to change this comment via the trello api, but this is not supported by the library so far.

**Comment by joneslee85** [comment]
So is there anyone up for this PR? I could provide mentorship if needed

**Comment by Larusso** [comment]
I have no time the next weeks.

**Comment by njt1982** [comment]
I dont believe the API supports getting comments on a card (cant see a relevant API endpoint https://developers.trello.com/advanced-reference/card#).

You **can** get actions (Trello:Card.find(x).actions). In the actions, you have the addComment type (as well as a few other relevant ones). So you could build your own list from that.

**Comment by andrewkress** [comment]
You can get the card comments with this call:
https://api.trello.com/1/cards/[card id or shortlink]/actions?filter=commentCard

**Comment by joneslee85** [comment]
Anyone up for this feature?
